project/ws/app/src/lib/routes/home/components/search/search.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, EventEmitter, Input, Output, OnInit } from '@angular/core'
+// import { ActivatedRoute } from '@angular/router'
+/* tslint:disable */
+import _ from 'lodash'
+/* tslint:enable */
+import { LoaderService } from '../../../../../../../../../src/app/services/loader.service'
+import { UsersService } from '../../../users/services/users.service'
+@Component({
+  selector: 'ws-app-searchuser',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss'],
+})
+export class SearchComponent implements OnInit {
+  @Input() from: any = ''
+  @Output() handleApiData = new EventEmitter()
+  searchText = ''
+  filterVisibilityFlag = false
+  clearFilter = false
+  designationList: any[] = []
```

Comment:
same as above

Reply:
Ya I have cleaned up the code, just added as I do the integration further I will clean up the code